### ISSUE TITLE
Remove tap2deb and tap2rpm

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8326]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8326|
|Type|task|
|Created|2016-05-10 16:48:11Z|
|Branch|https://github.com/twisted/twisted/tree/notap2deb-8326|

They've been deprecated long enough, and they're actively bad. Let's remove it.

<details><summary>Searchable metadata</summary>

```
trac-id__8326 8326
type__task task
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__notap2deb_8326 notap2deb-8326
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1462898891958398 1462898891958398
changetime__1463229027465401 1463229027465401
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

Yeah, I'm happy with this.

The ratcheted builder failues (pyflakes, apidocs, twistedchecker) are spurious; they're due to a "fresh" install of buildbot.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_3"></a>|@glyph commented|
|-|-|

This isn't a review of the code here per se, but it looks like buildbot is in a pretty raw state.  Can we get that addressed before landing anything?  I'd like to just see a wall of green ;)

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl commented|
|-|-|

There, I think I've got your desired wall of green, at least when the builders finish chewing through the backlog.

(That is, the buildbot list for this branch is flat green when the FBSD builder reruns)

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_5"></a>|@adiroiban set owner to @hawkowl|
|-|-|

Hooray. Great news.

Buildbot is a green wall :)

So now, we have a single file which does not pass the pylakes checkes.


please merge. thanks again!

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_6"></a>|@hawkowl set status to `closed`|
|-|-|

In changeset 55d0f4176c0ee4e23ad8ad28219de2f3d62f5d47
```
#!CommitTicketReference repository="" revision="55d0f4176c0ee4e23ad8ad28219de2f3d62f5d47"
Merge notap2deb-8326: Remove tap2deb and tap2rpm

Author: hawkowl
Reviewer: adiroiban
Fixes: #8326
```
